### ISSUE TITLE
feature request: report source of last operation

### ISSUE THREAD

**Issue by balk77** [CoMPaTech/esphome_c1101#3]
Hi, it would be nice to know what the source is off the last operation. Did it come from HA or did someone press the original button on the wall? I would like to have this to override the HA controlled speed and for that I would like to have a switch. Is that possible?

**Comment by CoMPaTech** [comment]
The 'HA or not-HA' is something that can easily be build by adding a virtual switch to flip - detecting which wall-remote was pressed I'm not sure about :) 
About the doable part, we could add/modify the behaviour of the esp to 'flip' a virtual switch (and/or input_binary) named 'last action by?' where 0 indicates HA and 1 indicates ESP -  to indicate who changed it last time around? Is that what you are looking for?

**Comment by balk77** [comment]
Yes, that would fullfil my needs :)
It does not matter which remote has been used. Would this work?
https://esphome.io/components/binary_sensor/template.html

**Comment by CoMPaTech** [comment]
Yeah that should work, i'll try to find some playing time coming weeks.
